lib/Myriad.pm

```diff
@@ -357,7 +357,7 @@ and metrics.
 method http () {
     unless($http) {
         $loop->add(
-            $http = Myriad::Transport::HTTP->new
+            $http = Myriad::Transport::HTTP->new(myriad => $self)
```

Comment:
The other components don't need a reference to `Myriad` why this one want?

Reply:
yeah I was trying to use RYU object. But I think its more logical to have its own rather than using this separate one